pkg/lib/util/file.go

```diff
@@ -17,14 +17,9 @@ var c = &http.Client{
 
 // OpenFileOrUrl opens either a local file or gives the resp.Body from a remote file
 func OpenFileOrUrl(path, rootDir string) (string, io.ReadCloser, error) {
-	if strings.HasPrefix(path, "@") {
-		path = path[1:]
-	} else if strings.HasPrefix(path, "p@") {
-		// p@ -> parallel tests
-		path = path[2:]
-	} else if IsParallelPathSpec(path) {
-		// pN@ -> N parallel repetitions of tests
-		_, path = GetParallelPathSpec(path)
+	pathSpec, err := ParsePathSpec(path)
+	if err == nil {
```

Comment:
Wir sollten den Error hier in "_" capturen und einen Comment reinschreiben dass, wenn die path spec falsch ist das schon vorher abgefangen wird.